doc/source/snippets/scikit-installation.rst

```diff
@@ -9,16 +9,16 @@ Install ``scikit-package`` with conda
 
     .. code-block:: bash
 
-        conda create -n skpkg_env scikit-package pre-commit
+        $ conda create -n skpkg_env scikit-package pre-commit
```

Comment:
I think we shall adopt `$` for bash scripts.. also found an extension that allows us to copy the code block.. will make a separate issue for this.